tasks/packages.yml

```diff
@@ -41,9 +38,8 @@
 
 - name: Install basic software
   package:
-    name: "{{ item }}"
-    state: installed
-  with_items: "{{ system_packages }}"
+    name: "{{ system_packages }}"
```

Comment:
`system_packages` is a list so it probably should be used in a loop.

Reply:
Loops are not necessary with module package,apt etc. - you can just pass a list. It was changed in ver 2.5 of ansible (not 100% sure about version but its was changed)

Reply:
No it wasn't. `package` module just passes info to next module, so in case of debian-based installations it will pass everything to `apt` (which accepts a list), but on other platforms it might fail.

Reply:
I might be wrong but looking at [it](https://docs.ansible.com/ansible/devel/porting_guides/porting_guide_2.7.html#deprecated) , I think it should be left as a list since squash_actions is going to be removed. Anyway tomorrow I'm going to check if this task generates "deprecated" warnings as in case of apt module.

Reply:
Seems legit and since this role officially supports only ubuntu xenial and CentOS7 then there is no use in checking edge cases like Arch Linux etc.